src/plugins/image/editing/image-editor.js

```diff
@@ -15,11 +16,15 @@ const RadioGroup = Radio.Group;
 const FormItem = Form.Item;
 const { TextArea } = Input;
 
+const defaultClipRegion = { x: 0, y: 0, width: 0, height: 0, data: {} };
```

Comment:
One suggestion and on question:

* Wouldn't it be nicer from a UX perspective to set the default to `{ x: 0, y: 0, width: 100, height: 100 }`? It's hard to spot the little square in the left top corner after updating the image source.
* `data` seems to be always an empty object, or did I miss something?

Reply:
- It's a good idea, we can set it to a default rectangle that is actually visible. 
Since the values are percentages, shall we go for something like `{ x: 0, y: 0, width: 15, height: 15 }` ?
- Yes, `data` is for us an empty object always, the plugin does need it to be present though

Reply:
As I consider "clipping" to be a process started from the original size, I'd go for 100, but on the other hand the rectangle would be more visible if it were inset by 10% of each side. Your call :-)

Reply:
Can we just feed the `data` property to the plugin, but not persist it as part of the data model?

Reply:
- Went for the "10% on each size" suggestion, I think that's a nice one. "100%" would have been just as invisible as "0%" I think, hard for the user to notice the rectangle.
- Indeed, I thought of `data` as the model of the plugin, what it needs to render correctly, but we don't need it in out own persisted model. Removed, thanks!

Reply:
Cool, thanks!